File: src/Web.API/Models/Identity/UpdateUser/UpdateUserRequest.cs
using System;

namespace Web.API.Models.Identity.UpdateUser
{
	public class UpdateUserRequest
	{
		public string UserName { get; set; }
		public string NewFirstName { get; set; }
		public string NewLastName { get; set; }
		public string? NewPhoneNumber { get; set; }
		public bool RemoveImage { get; set; }
	}
}

File: src/Web.Angular/ClientApp/src/app/DTOs/Identity/UpdateUserRequest.ts
export class UpdateUserRequest {

  public userName: string;
  public newFirstName: string;
  public newLastName: string;
  public newPhoneNumber: string;
  public newImageFile: File | null = null;
  public removeImage: boolean = false;

  constructor(userName: string = "", newFirstName: string = "", newLastName: string = "", newPhoneNumber: string = "") {
    this.userName = userName;
    this.newFirstName = newFirstName;
    this.newLastName = newLastName;
    this.newPhoneNumber = newPhoneNumber;
  }
}
